models/endpoint.model.js

```diff
@@ -0,0 +1,8 @@
+const fs = require("fs/promises");
+
+exports.selectEndpoints = () => {
+  return fs.readFile(
+    "/home/leopold-harvey/northcoders/backend/review/be-nc-news/endpoints.json",
+    "utf-8"
+  );
```

Comment:
this file path works - but only on your computer. If you were hosting this server elsewhere, this file path wouldn't work! You should make this a bit more dynamic by applying ${__dirname} which represents the directory of the executing file.

Also, this functionality isn't a database query, so it doesn't need to be a model. Since we're only reading one local file, this endpoint doesn't need a model - just a controller.